components/odh-notebook-controller/controllers/notebook_oauth.go

```diff
@@ -35,7 +35,10 @@ import (
 const (
 	OAuthServicePort     = 443
 	OAuthServicePortName = "oauth-proxy"
-	OAuthProxyImage      = "registry.redhat.io/openshift4/ose-oauth-proxy:latest"
+	// OAuthProxyImage uses sha256 manifest list digest value of v4.8 image for AMD64 as default to be compatible with imagePullPolicy: IfNotPresent, overridable
+	// taken from https://catalog.redhat.com/software/containers/openshift4/ose-oauth-proxy/5cdb2133bed8bd5717d5ae64?image=6306f12280cc9b3291272668&architecture=amd64&container-tabs=overview
+	// and kept in sync with the manifests here and in ClusterServiceVersion metadata of opendatahub operator
+	OAuthProxyImage = "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46"
```

Comment:
I'm unsure whether to leave it as :latest here and only set hash through the manifest. It seems sensible to me to leave :latest in the code.

Reply:
Agreed, basically, code here to a floating type tag like latest is best for development environment.

Since the value is overriden in manifest manager.yaml with the custom argument containing the hash, we can leave it to "latest" here.

Reply:
I've changed my mind. I feel more comfortable with having the digest here, and also hardcoding the PullIfNotPresent. There's not much of real value in testing with `:latest`, anyways. Seems to me that using :latest oauth proxy image  is almost always an error.

(It does make some sense to have the notebook controller image in :latest, but that's a different question)

For now, I'll keep the PR without making the changes I considered here.

Reply:
I agree with @jiridanek on this, i.e. having the digest here as well, and also hardcoding the PullIfNotPresent

Reply:
We'll be going with 4.14 version of the image, https://github.com/opendatahub-io/kubeflow/pull/386

Reply:
yup, nice